libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -548,6 +549,19 @@ public void setContent(CharSequence text) {
         mAztecReady = true;
     }
 
+    private void addAztecAudioShortcodePlugin() {
+        boolean foundAudioShortCodePlugin = false;
+        for (IAztecPlugin plugin : mContent.getPlugins()) {
+            if (plugin instanceof AudioShortcodePlugin) {
+                foundAudioShortCodePlugin = true;
```

Comment:
Niptick: You can just `return` here, and avoid to use `boolean foundAudioShortCodePlugin`

Reply:
nice, addressed in a1d549cff6